traffic_portal/app/src/common/modules/table/serviceCategoryDeliveryServices/table.serviceCategoryDeliveryServices.tpl.html

```diff
@@ -26,7 +26,6 @@
 		columns="columns"
 		drop-down-options="dropDownOptions"
 		context-menu-options="contextMenuOptions"
-		sensitive-columns="sensitiveColumns"
```

Comment:
On the service category/delivery services page I can now see all the content of sensitive columns, which I think is because you removed this.

Reply:
`sensitiveColumns`is not defined on these child tables or on the parent `TableDeliveryServiceController`

Reply:
I don't know how that slipped through the cracks...